01/app.js

```diff
@@ -1,2 +1,8 @@
 const randomNumber = Math.round(Math.random() * 20);
 console.log(randomNumber);
+
+if (randomNumber > 5) {
+  for (let i = 5; i <= randomNumber; i++) console.log(i);
```

Comment:
Czytelniej jest zawsze pisac nawiasy `{ ... }` do pętli `for` (jak i każdej innej :P)